src/Tasks/AssemblyDependency/GenerateBindingRedirects.cs

```diff
@@ -335,7 +336,12 @@ private XDocument LoadAppConfig(ITaskItem appConfigItem)
             }
             else
             {
-                document = XDocument.Load(appConfigItem.ItemSpec);
```

Comment:
Can you ban this API in `src\BannedSymbols.txt` so we never accidentally reintroduce it?

Reply:
done